css/main.css

```diff
@@ -0,0 +1,347 @@
+body{
+	font-style: normal;
+	font-family: Arial;
+	margin: 0;
+	min-width: 320px;
+}
+
+.main-header{
+	height: 97px;
+	font-size:12px;
+	display: flex;
+	text-transform: uppercase;
+	font-weight: 400;
+	background-color: #373c3f;
+}
+
+.main-header__ul{
+	height: 12px;
+	display: inline-flex;
+	flex-wrap: wrap;
+	list-style: none;
+	margin:45px 0 0 54px;
+	padding: 0;
+}
+
+.main-header__li--screen-size--full{
+	height: 12px;
+	padding: 0 1.2rem;
+}
+
+.main-header__li--screen-size--low{
+	display: none;
+}
+
+.main-header__a{
+	color:#ebebeb;
+	text-decoration: none;
+}
+
+.main-header__menu{
+	list-style: none;
+}
+
+.intro-content{
+	background: linear-gradient(#19b4c6, #078aa4);
+	padding: 0;
+	height: 578px;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.intro-content__fisrtheadline{
+	font-size: 50px;
+	color: #ffffff;
+	font-size: 58px;
+	margin-left: 80px;
+}
+
+.intro-content__arrow__link{
+	font-size: 32px;
+	color: #a7d4de;
+	text-decoration: none;
+	margin-left: 80px;	
+}
+
+.intro-content__arrow{
+	display: flex;
+	align-items: center;
+}
+
+.intro-content__arrow-right{
+	border: solid white;
+	border-width: 0 4px 4px 0;
+	display: inline-block;
+	padding: 7px;
+	transform: rotate(-45deg);
+	border-radius: 2px;
+	margin-left: 30%;
+}
+
+.robin-hood{
+	height: 245px;
+	padding: 0;
+	background-color: #27212c;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+}
+
+.robin-hood__paragraph{
+	font-size: 32px;
+	margin: auto;
+	color: #9282a0;
+}
+
+.robin-hood__arrow{
+	background-color: #bc3183;
+	width: 25%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+.robin-hood__arrow__arrow--right{
```

Comment:
I guess it's better to keep just ".robin-hood__arrow--right". Then you can apply this class like:
`<div class="robin-hood__arrow     robin-hood__arrow--right"> ... </div>`